### PR TITLE
fix(curriculum): update description of if statement

### LINKED ISSUE #53429: Update if statement explanation for RPG game step 76
### Describe the Issue

Right now, we have two steps dedicated to teaching an `if` statement.
The first step has campers move all of the function code inside an if statement with the string `condition` for the condition.
https://www.freecodecamp.org/learn/javascript-algorithms-and-data-structures-v8/learn-basic-javascript-by-building-a-role-playing-game/step-76
Then the next step, has them change the string `condition` to the condition `gold >=10`.
https://www.freecodecamp.org/learn/javascript-algorithms-and-data-structures-v8/learn-basic-javascript-by-building-a-role-playing-game/step-77

I think this can be simplified and combined into just one step.
Also, I think this will give us a better chance to explain what an if statement is and how the condition works. 

## proposed new directions

````md
# --description--

What if the player doesn't have enough gold to buy health? When you want to run code conditionally, you can use the <dfn>if</dfn> statement.

An `if` statement is used to make decisions in code. The keyword `if` is followed by a condition in parentheses. If the condition is true, the code inside the curly braces `{}` is executed. If the condition is false, the code inside the curly braces is skipped.

Here is an example of an `if` statement:

```js
const num = 5;
if (num >= 3) {
  console.log("This code will run because num is greater than or equal to 3.");
}
```

Start by placing all of the code in your `buyHealth` function inside an `if` statement. For the `if` statement condition,  check if `gold` is greater than or equal to `10`.
````
thoughts?

### Affected Page

multiple urls

### Your code

see explanation above

### Expected behavior

see explanation above

### Screenshots

_No response_

### System

- Device: [e.g. iPhone 6, Laptop]
- OS: [e.g. iOS 14, Windows 10, Ubuntu 20.04]
- Browser: [e.g. Chrome, Safari]
- Version: [e.g. 22]

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by netanelben**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #53429

### CODE REVIEW

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Hi @netanelben ! 

There are two things missing from this PR.

Updating the description is only part of the issue.
Since the description is updated, that means the tests need to be updated to. Right now your tests are still checking for the old description. Please make to appropriate changes to the tests so they match the current new description. 

Once you resolve that, you will also need to get rid of the next step. Currently the next step has them adding the condition for the if statement. 
But since we updated the description for this step to include that, there is no need for the next step. 
Please remove it

File: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-role-playing-game/62a8c4db0710f3260f867a92.md [hunk 1]
```diff
@@ -7,17 +7,20 @@ dashedName: step-76
...
+const num = 5;
+if (num >= 3) {
+  console.log("This code will run because num is greater than or equal to 3.");
 }
 ```
 
-For now, the `if` statement condition should be set to the string `condition` as a placeholder.
+Start by placing all of the code in your `buyHealth` function inside an `if` statement. For the `if` statement condition,  check if `gold` is greater than or equal to `10`.
 
 # --hints--
```
Comment (JoyShaheb) [inline curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-role-playing-game/62a8c4db0710f3260f867a92.md:26]: @netanelben i got you buddy, try these hints i wrote for you. Also, check the contributing guides, you should find some instructions on how to delete the step number 77 using the terminal[if you're on windows, believe me, its gonna be a huge pain!]. If you get stuck, reply here I'll be more than happy to help ! Also @jdwilkin4  what do you think of the solution ?

# --hints--

Your `if` statement should check if `gold` is greater than or equal to `10`.

```js
assert.match(buyHealth.toString(), /if\s*\(\s*gold\s*>=\s*10\s*\)/);
```

All of your `buyHealth` code should be inside the `if` statement. If you've deleted the original code, you can reset the step and try again.

```js
assert.match(buyHealth.toString(), /if\s*\(\s*gold\s*>=\s*10\s*\)\s*{\s*gold\s*-=\s*10;\s*health\s*\+=\s*10;\s*goldText\.innerText\s*=\s*gold;\s*healthText\.innerText\s*=\s*health;\s*}/);
```

**Review by JoyShaheb — CHANGES_REQUESTED** [review]

File: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-role-playing-game/62a8c4db0710f3260f867a92.md (hunk 1 above)
Comment (netanelben) [reply curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-role-playing-game/62a8c4db0710f3260f867a92.md:26]: thanks @JoyShaheb i am looking into that right now. I am using mac of-curse ^_^

Comment (JoyShaheb) [reply curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-role-playing-game/62a8c4db0710f3260f867a92.md:26]: @netanelben cheers buddy 🥂[meanwhile I'm stuck with windows :')]

Comment (netanelben) [reply curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-role-playing-game/62a8c4db0710f3260f867a92.md:26]: I need some assistant here 
I'm trying to run the `delete-step` script and getting this err:
```Error: ENOENT: no such file or directory, open '/Users/nate/_meta/curriculum/meta.json'```

Comment (jdwilkin4) [reply curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-role-playing-game/62a8c4db0710f3260f867a92.md:26]: Are you running the script in the correct directory?

If you are coming from the root directory, open up a new tab in the terminal.
Then run this command to get to the correct directory ` cd curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-role-playing-game/`

then run the delete command `pnpm run delete-step 77`

Comment (jdwilkin4) [reply curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-role-playing-game/62a8c4db0710f3260f867a92.md:26]: We can remove this part here from the hint text

> If you've deleted the original code, you can reset the step and try again.

File: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-role-playing-game/62a8c4db0710f3260f867a92.md [hunk 2]
```diff
@@ -30,13 +33,13 @@ assert.match(buyHealth.toString(), /if/);
 Your `if` statement should have the string `condition` for the condition.
```
Comment (jdwilkin4) [inline curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-role-playing-game/62a8c4db0710f3260f867a92.md:33]: this hint text needs to change. We shouldn't check for the string condition. It should check for gold is greater than or equal to 10

**Review by jdwilkin4 — APPROVED** [review]

**Review by JoyShaheb — APPROVED** [review]
Well done buddy 🥂

**Pull request merged**